qcodes/instrument/parameter.py

```diff
@@ -339,27 +336,11 @@ def raw_value(self) -> ParamRawDataType:
         :setter: Setting the ``raw_value`` is not recommended as it may lead to
             inconsistent state of the parameter.
         """
-        return self._get_cache_raw()
+        return self.cache._raw_value
 
     @raw_value.setter
     def raw_value(self, new_raw_value: ParamRawDataType) -> None:
```

Comment:
indeed, the idea is to deprecate-and-remove `_BaseParameter.raw_value` altogether in the future. since last release `raw_value` has been converted from an attribute to a property so that a more potentially harmful setting of the `raw_value` can be deprecated/removed first, and then later removed altogether thanks to introduction of `.cache` with all it's "greatness".

The docstring of the `raw_value` property already notes that it will be deprecated soon and that setting it is dangerous.